erase-install/erase-install.munki.recipe

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads the latest version of the erase-install script and imports into Munki.</string>
+    <key>Identifier</key>
+    <string>com.github.its-unibas.munki.erase-install</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>erase-install</string>
+        <key>MUNKI_REPO_SUBDIR</key>
+        <string>apps/%NAME%</string>
+        <key>pkginfo</key>
+        <dict>
+            <key>catalogs</key>
+            <array>
+                <string>testing</string>
+            </array>
+            <key>category</key>
+            <string>Productivity</string>
+            <key>description</key>
+            <string></string> 
```

Comment:
Add a description. How about:
"A script that automates downloading macOS installers and erasing or upgrading macOS in a single process. "